src/main/java/net/mcreator/ui/browser/action/NewFolderAction.java

```diff
@@ -37,12 +37,11 @@
 	public NewFolderAction(ActionRegistry actionRegistry) {
 		super(actionRegistry, L10N.t("action.browser.new_folder"), actionEvent -> {
 			String foldername = VOptionPane.showInputDialog(actionRegistry.getMCreator(),
-					L10N.t("workspace_file_browser.new_folder_name.folder_name"),
-					L10N.t("workspace_file_browser.new_folder_name.folder_name.title"), null,
-					new OptionPaneValidatior() {
+					L10N.t("workspace_file_browser.new_folder.folder_name"),

```

Comment:
Please do not rename keys when not needed as this will remove existing translations by users of crowdin